cfme/intelligence/reports/reports.py

```diff
@@ -106,7 +106,7 @@ def is_displayed(self):
             self.reports.is_opened and
             self.reports.tree.currently_selected == [
                 "All Reports",
-                "My Company (All EVM Groups)",
+                mycompany_title(),
```

Comment:
**REQUIRED**
I would add a property in `CloudIntelReportsView`:
```python
@property
def mycompany_title(self):
    if self.browser.product_version < "5.9":
        title = "My Company (All EVM Groups)"
    else:
        title = "My Company (All Groups)"
    return title
```
And further you can write like this:
```python
self.reports.tree.currently_selected == ["All Reports", self.mycompany_title ...
```

Reply:
@quarckster , Done